modules/web-console/frontend/app/components/page-queries/components/queries-notebook/controller.ts

```diff
@@ -2147,10 +2149,10 @@ export class NotebookCtrl {
             }
         };
 
-        const offTransitions = $transitions.onBefore({from: 'base.sql.notebook'}, ($transition$) => {
+        this.offTransitions = $transitions.onBefore({from: 'base.sql.notebook'}, ($transition$) => {
             const options = $transition$.options();
 
-            if (options.custom.justIDUpdate || options.redirectedFrom)
+            if (options.redirectedFrom)
```

Comment:
Please add comment as to why `redirectedFrom` matters.

Reply:
Added comment.